example/ios/Podfile.lock

```diff
@@ -393,7 +393,7 @@ PODS:
     - ReSwiftThunk/Core (= 2.0.1)
   - ReSwiftThunk/Core (2.0.1):
     - ReSwift (~> 6.0)
-  - Rownd (2.8.3):
+  - Rownd (2.9.0):
```

Comment:
Were we going to do a major version bump?

Reply:
I think we wanted to do a major version bump because of the sodium package removal

Reply:
Major bump would be 3.0.0

Reply:
Updated